### PR TITLE
Woodcutting

### PR DESCRIPTION
**Pull request by coolbop32**
Added the woodcutting Skill

### CODE REVIEW

File: src/commands/Minion/chop.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+		// Calculate the time it takes to mine a single log of this type, at this persons level.
+		let timetoChop = determineScaledLogTime(
+			log!.xp,
+			log.respawnTime,
+			msg.author.skillLevel(SkillsEnum.Woodcutting)
+		);
+
+		// If the user has a dragon axe & over 61 mining provide 10% speed boost
+		if (
```
Comment (gc) [inline src/commands/Minion/chop.ts]: change this how to we check for mining pickaxes, also in line with what we do for mining, give infernal a higher boost.

File: src/commands/Minion/chop.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+			(await msg.author.hasItem(itemID('Dragon axe'), 1, true)) ||
+			((await msg.author.hasItem(itemID('Infernal axe'), 1, true)) &&
+				msg.author.skillLevel(SkillsEnum.Mining) > 61)
```
Comment (gc) [inline src/commands/Minion/chop.ts]: checking for mining? this is woodcutting

File: src/tasks/minions/woodcuttingActivity.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+		if (Log.petChance && rand(1, Log.petChance * 1.5) < quantity) {
+			loot[WoodcuttingPet!.id] = 1;
+			str += `\nYou have a funny feeling you're being followed...`;
+		}
+
+		str += `\n\nYou received: ${await createReadableItemListFromBank(this.client, loot)}.`;
+
+		await user.addItemsToBank(loot, true);
+
+		let channel = this.client.channels.get(channelID);
```
Comment (gc) [inline src/tasks/minions/woodcuttingActivity.ts]: see the other tasks for how to change this, look for `channelIsSendable`

File: src/tasks/minions/woodcuttingActivity.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+				const response = messages.first();
+
+				if (response) {
+					user.log(`continued trip of ${quantity}x ${Log.name}[${Log.id}]`);
+					this.client.commands
+						.get('chop')!
+						.run(response as KlasaMessage, [quantity, Log.name]);
+				}
+			})
+			.catch(noOp);
```
Comment (gc) [inline src/tasks/minions/woodcuttingActivity.ts]: wrap this in `this.client.queuePromise`, see monsterActivity.ts.

**Comment by coolbop32** [comment]
I think the issue is my version does not have quest points in it

**Comment by gc** [comment]
I think this PR was rebased incorrectly, all my commits have been replayed over it (and showing in the changed files).

**Pull request closed without merging**